pieforproviders/pieforproviders
# Adds new endpoint for the dashboard, updates it to return case_number and child full_name

Adds new endpoint for the dashboard, updates it to return case_number and child full_name

# 💅🏼 What issue does this fix?
#481 and #463 with a new endpoint implementation

## 🍂 Before You Submit


## 🧵 Steps to set up locally

- Might need to restart your rails server to pick up the new route

## 🧳 Steps to test

- login via Postman
- use the token from the Authorization header in the login response in your request Authorization header as a bearer token
- hit the new endpoint: `/api/v1/case_list_for_dashboard`
- you should see the following response format, if your user has any children associated with it:
```
[
    {
        "full_name": "Maria Baca",
        "approvals": [
            {
                "case_number": "5184465331"
            }
        ]
    },
    {
        "full_name": "K'Shawn Henderson",
        "approvals": [
            {
                "case_number": "5198908053"
            }
        ]
    },
    {
        "full_name": "Marcus Smith",
        "approvals": [
            {
                "case_number": "9372391607"
            }
        ]
    },
    {
        "full_name": "Sabina Akers",
        "approvals": [
            {
                "case_number": "9907008287"
            }
        ]
    },
    {
        "full_name": "Mubiru Karstensen",
        "approvals": [
            {
                "case_number": "1525425994"
            }
        ]
    },
    {
        "full_name": "Tarquinius Kelly",
        "approvals": [
            {
                "case_number": "3438371868"
            }
        ]
    }
]
```
